content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

```diff
@@ -123,6 +123,14 @@ $ kubectl edit clusterrole system:kube-scheduler
     - update
 ```
 
+`my-scheduler` runs as the `default` service account in the `kube-system` namespace. Bind cluster role `system:kube-scheduler` to allow access to the resources required.
+
+``` shell
+kubectl create clusterrolebinding my-scheduler-kube-scheduler \
+  --clusterrole=system:kube-scheduler \
+  --serviceaccount=kube-system:default
```

Comment:
This could work but ... it is creating another security issue. I'm not a fan of changing the built in accounts. Instead I'd suggest we create a new service account for this.

Reply:
Fixed and tested locally. @tengqm